.vscode/launch.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
do you need this in the PR ?

Reply:
No but it is surprising the file is not there yet. We usually have it everywhere.

Reply:
so....can you delete it ?

Reply:
Removed.
I do not necessarily agree, the majority of us uses VS Code, this is so convenient to have debugging env ready on project checkout...